react-native/.circleci/config.yml

```diff
@@ -0,0 +1,351 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.1.0
+
+references:
+  workspace: &workspace ~/project
+
+  workspace_android: &workspace_android ~/project/android
+
+  workspace_ios: &workspace_ios ~/project/ios
+
+  android_build_outputs: &android_build_outputs ~/project/android/app/build/outputs
+  ios_build_outputs: &ios_build_outputs ~/project/ios/builds
+
+  # Configs
+  node_config: &node_config
+    working_directory: *workspace
+    docker:
+      - image: cimg/node:20.10.0
+      - image: cimg/ruby:3.0.0
+
+  android_config: &android_config
+    working_directory: *workspace_android
+    docker:
+      - image: cimg/android:2023.10.2-browsers
+    resource_class: xlarge
+
+  ios_config: &ios_config
+    working_directory: *workspace_ios
+    macos:
+      xcode: 15.4.0
```

Comment:
please, add a todo comment
'need to adjust to actual version'
also in other places where some versions are defined in this config

Reply:
Done